xray/plot/facetgrid.py

```diff
@@ -127,14 +129,21 @@ def __init__(self, data, col=None, row=None, col_wrap=None,
                 ncol = col_wrap
             nrow = int(np.ceil(nfacet / ncol))
 
+        # Set the subplot kwargs
+        subplot_kws = {} if subplot_kws is None else subplot_kws.copy()
+
         # Calculate the base figure size with extra horizontal space for a
         # colorbar
         cbar_space = 1
-        figsize = (ncol * size * aspect + cbar_space, nrow * size)
+        figsize = subplot_kws.pop('figsize', (ncol * size * aspect + cbar_space,
+                                              nrow * size))
+        sharex = subplot_kws.pop('sharex', True)
+        sharey = subplot_kws.pop('sharey', True)
+        squeeze = subplot_kws.pop('squeeze', False)
```

Comment:
Turns out I was misreading this section of the code and it can go back to the way it was.